static/app/components/searchQueryBuilder/tokens/filter/aggregateKey.tsx

```diff
@@ -28,7 +29,7 @@ type AggregateKeyProps = {
 
 export function AggregateKeyVisual({token}: {token: AggregateFilter}) {
   const fnName = getKeyName(token.key);
-  const fnParams = token.key.args?.text ?? '';
+  const fnParams = prettifyTagKey(token.key.args?.text ?? '');
```

Comment:
This doesn't look right - this is for filters that look like `count_if(transaction.duration,equals,300):>1`